.github/workflows/deploy.yml

```diff
@@ -36,7 +36,9 @@ jobs:
         run: yarn install --frozen-lockfile
 
       - name: Build pages
-        run: yarn build
+        run: |
+          export PUBLIC_URL=/FOSSologyUI
```

Comment:
Please use the `env` format: https://docs.github.com/en/actions/reference/environment-variables

```yaml
env:
  PUBLI_URL: /FOSSologyUI
```

Reply:
Also I think the `.env` file is not created from `.env.example`. Please add that step too.

Reply:
Okay @GMishx